docs/src/proposals/abi-v2.md

```diff
@@ -0,0 +1,88 @@
+---
+title: ABI v2
+---
+
+ABI between loader and program entrypoint, as well as syscalls such as (cross program) invocation and account reallocation.
```

Comment:
I don't see anything syscall related in this document

Reply:
Right, will add a section for them as well.